infra-as-code/bicep/modules/hubNetworking/hubNetworking.bicep

```diff
@@ -76,7 +76,15 @@ param parAzFirewallTier string = 'Standard'
   '3'
 ])
 @description('Availability Zones to deploy the Azure Firewall across. Region must support Availability Zones to use. If it does not then leave empty.')
-param parAzFirewallAvailabilityZones array = []
+param parAzFirewallAvailabilityZones array 
+
+@allowed([
+  '1'
+  '2'
+  '3'
+])
+@description('Availability Zones to deploy the VPN/ER PIP across. Region must support Availability Zones to use. If it does not then leave empty. Ensure that you select a zonal SKU for the ER/VPN Gateway if using Availability Zones for the PIP')
+param parAzGatewayAvailabilityZones array
```

Comment:
Instead of introducing this as a parameter that is applied to both VPN and ER GWs regardless.

Could we instead add this as a new key to each of the following parameters with the name of `zones` (as an example):
- `parExpressRouteGatewayConfig`
- `parVpnGatewayConfig`

Then this gives the flexibility to configure say the VPN GW without zones and the ER GW with zones (if a customer so chooses).

Reply:
Done. Needs testing

Reply:
tested. will discuss tomorrow